index.html

```diff
@@ -46,48 +46,8 @@
 	<!-- projects -->
 	<div class="projects">
 		<div class="container">	
-			<h1>My Projects</h1>	
```

Comment:
If you use a `<div class="container">...</div>` you must use `row` && `col-xxx-xxx` classes

```css
<div class="container">
  <div class="row">
    <div class="col-xxx-xxx">
      Your content here.
    </div>
  </div>
</div>
```